### PR TITLE
Initial files.

### PR DESCRIPTION
**Pull request by davidlehn**
This is a starting point based on https://github.com/digitalbazaar/vc-status-list with renamed properties and simple updates.  Some tests pass, some fail, some probably pass because of unrelated failures are caught, some seem to fail due to invalid mock data signatures.  An incomplete list of items to address:

- [x] Fix current tests.
- [ ] Check tests match the vc-bitstring-status-list spec vs the vc-status-list[-2021] one.
- [ ] Add more tests to cover new features and properties (`ttl`, `statusReference`, `stsatusSize`, `statusMessage`, etc).
- [x] Mock `encodedList` data needs to be multibase formatted.
- [x] Add tests to catch non-multibase `encodedList`.
- [x] Mock data likely needs to be re-generated to get valid new signatures.  (I'm not sure where this data came from or if a script exists to do this.)
- [x] The tests are based on VC V1 context.  In theory the context should be compatible with V2, which should likely be tested, even though the context is redundant.

If there are errors related to search and replace from vc-status-list, mass replacing of the changes is fine.  May be worth comparing the files to see what was changed.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+ *
+ * @param {object} options - Options to use.
+ * @param {string} options.id - The id for BitstringStatusList Credential.
+ * @param {BitstringStatusList} options.list - An instance of
+ *   BitstringStatusList.
+ * @param {string} options.statusPurpose - The purpose of the status entry.
+ *
+ * @returns {object} The resulting `BitstringStatusList Credential`.
+ */
+export async function createCredential({id, list, statusPurpose}) {
```
Comment (brianorwhatever) [inline lib/index.js]: @dlongley what approach should I take for having this function able to create credentials in both VCDM 1 and 2? I've considered adding a context argument directly or a `vcdmVersion` (or something similar) argument

Comment (dlongley) [reply lib/index.js]: @brianorwhatever, I guess a `context` argument is the way to go -- and then check to make sure it's either v1 or v2 (against a supported list that we could further expand in the future). The `context` param can default to v2.

Comment (brianorwhatever) [reply lib/index.js]: @davidlehn do you have thoughts on the above?

Comment (davidlehn) [reply lib/index.js]: A param of some sort seems fine.  Would be more flexible going forward vs, say, another function.  To be clear, the suggestion here is to overload a context URL as a version string via a `context` param?  I find that awkward at best, and it's done elsewhere too, and I could rant a while on it.  Using a param with a more readable name and a value of a spec short name "vc-data-model" "vc-data-model-2.0" is an option too.  As is "1.0" "1.1" "2.0", etc, depending on the expectation of the usage here.

Comment (brianorwhatever) [reply lib/index.js]: hm sorry missed your reply above. Sounds good I will add a context param

**Review by dlongley — APPROVED** [review]
Approving to move this along. We don't need the extra multistatus features at this time and if we need anything else we can do another release.

**Pull request merged**